.github/workflows/image-build-and-push-community.yaml

```diff
@@ -19,7 +19,7 @@ on:
     #        │  │  │ │ │
     #        │  │  │ │ │
     #        │  │  │ │ │
-    - cron: 30 19 * * 0
+    # - cron: 30 19 * * 0
```

Comment:
is this temporary? if not we can remove rather than commenting out

Reply:
we can remove instead